ca/tls.go

```diff
@@ -139,7 +169,7 @@ func IntermediateCertificate(sign *api.SignResponse) (*x509.Certificate, error)
 
 // RootCertificate returns the root certificate from the sign response.
 func RootCertificate(sign *api.SignResponse) (*x509.Certificate, error) {
-	if sign.TLS == nil || len(sign.TLS.VerifiedChains) == 0 {
+	if sign == nil || sign.TLS == nil || len(sign.TLS.VerifiedChains) == 0 {
 		return nil, errors.New("ca: certificate does not exists")
```

Comment:
certificate does not exist$ (not exists)

Reply:
Fixed with  [f1f6c54](/smallstep/certificates/pull/27/commits/f1f6c548ad2eaf7338393cf964938adb8d4b0295)